src/pages/Integrations/List/useIntegrationRows.ts

```diff
@@ -76,11 +76,15 @@ export const useIntegrationRows = (integrations: Array<UserIntegration>) => {
                 limit.clearQueue();
 
                 integrations.map(integration => integration.id).forEach(integrationId => {
-                    limit(() => query(listIntegrationHistoryActionCreator(integrationId))).then(response => {
+                    limit(() => query(listIntegrationHistoryActionCreator({
+                        integrationId,
+                        limit: 5,
+                        sortBy: 'invocation_time:desc'
```

Comment:
@gwenneg Is this the expected sorting? or it should be created?

Reply:
Oops I was looking at the policiies history earlier and mixed everything 😄

`invocation_time` will contain a duration, so I wouldn't sort on that. I think `created:desc` is the good way to sort history entries.

Reply:
Ok I'll do it in the existing backend PR.

Reply:
The REST API is tightly coupled with the Hibernate layer because of the `Query` class. That's the reason why I never use that class in new APIs. I think we should use `nh.created:desc` without documenting it for now and get rid of the layers coupling in a subsequent PR. Pushing the limit to prod should be done ASAP because of the prod incident we had recently.